test/forms/Validators.test.js

```diff
@@ -244,6 +245,100 @@ describe('Validators', () => {
         });
     });
 
+    describe('isPositiveNumber', () => {
+        it('should return true when the value is 100', () => {
+            expect(isPositiveNumber(100)).to.be.true;
+        });
+
+        it('should return true when the value is the string "100"', () => {
+            expect(isPositiveNumber('100')).to.be.true;
+        });
+
+        it('should return true when the value is 1.9', () => {
+            expect(isPositiveNumber(1.9)).to.be.true;
+        });
+
+        it('should return true when the value is the string "1.9"', () => {
+            expect(isPositiveNumber('1.9')).to.be.true;
+        });
+
+        it('should return true when the value is -1.9', () => {
```

Comment:
@nicolayr 
Ah! That was too silly of me. :( 
I have fixed this in my commit : https://github.com/adhbh/d2-ui/commit/ab1c75a7b0afe8d985f4e6758ef54847d1473c84